### PR TITLE
Basic-structure added - different areas of the website

### PR DESCRIPTION
**Pull request by MichiBerger**
First I´ve tried to structure my website in different areas like the wirefram and I´ve added some comments in the code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,13 +21,31 @@ <h1>Michi Berger</h1>
...
+
+  <!-- skills section - e.g. skills (dev tools, dev languages), hobbies or interests -->
+  <section id="skills"></section>
+
+  <!-- contact section - address, email -->
   <section id="contact">
     <h2>Contact</h2>
   </section>
+
+  <!-- recent-work section - github, codepen,  -->
```
Comment (JelenadlS) [inline index.html:33]: Nice idea with the comments and info what still needs to be entered :-)

File: index.html [hunk 2]
```diff
@@ -21,13 +21,31 @@ <h1>Michi Berger</h1>
...
+  <!-- about-me section - short intro about me -->
+  <section id="about-me"></section>
+
+  <!-- biography section - working experience, education -->
   <section id="biography">
     <h2>Biography</h2>
   </section>
+
+  <!-- footer and social-media section e.g. linkedIn, instagram, facebook -->
+  <footer id="footer">
```
Comment (JelenadlS) [inline index.html:45]: not sure if the footer needs to have the id. But I think you have more knowledge, so it might be right :-)

**Review by JelenadlS — COMMENTED** [review]
Looks really good and very well structured 👍🏼

File: index.html (hunk 2 above)
Comment (teifler) [reply index.html:45]: > 
Well written code in general and nice with the comments! I think you dont need the ID but you can use it to style the footer with css later on.

**Review by teifler — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (MichiBerger) [reply index.html:45]: thx Jelena for the review :) actually I´m not sure about the id :D I think it´s not wrong to place one...maybe with a better "semantic" name

**Pull request merged**